src/lib/viewers/controls/annotations/AnnotationsButton.tsx

```diff
@@ -11,6 +13,8 @@ export type Props = Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'onClick
     isEnabled?: boolean;
     mode: AnnotationMode;
     onClick?: (mode: AnnotationMode) => void;
+    setWasClosedByUser?: (experienceName: string | undefined) => void;
+    tooltipFlowAnnotationsExperience?: TargetingApi;
 };
 
 export default function AnnotationsButton({
```

Comment:
Should every annotation button be wrapped in this tooltip? Will only one show?

Reply:
My first way of doing it was every AnnotationsButton was wrapped in this tooltip, but it was only ever shown for one due to the shouldTarget prop. I've moved the tooltip out to the Annotations Control component. Now it only wraps one button. Let me know what you think. I wasn't sure if creating another component was a good idea. Thank you!